install.sh

```diff
@@ -77,15 +67,14 @@ function install_dependencies()
     then
         if ! [ -x "$(command -v swig)" ]
         then
-            install_swig_from_sources
+            echo "swig not installed"
         else
             installed_version=$(swig -version | perl -nae 'print "$F[2]\n" if /SWIG Version/i;')
             minimum_version="3.0.10"
             result=$(vercomp ${installed_version} ${minimum_version})
```

Comment:
Why do you have to compare the version here?
SWIG is installed from OS repo, it cannot have version lower than the one in repo. And we don't support Debian older than 9, so the Swig won't be older than 3.0.12.
No need to compare here.